### ISSUE TITLE
Email field issue on Instagram when Relay add-on is installed

### ISSUE THREAD

**Issue by Bianca-O** [mozilla/fx-private-relay-add-on#109]
**Build:**
Stage

**Affected platforms:**
Windows 10 64-bit

**Browsers tested on:**
Firefox

**Preconditions:**
The user is logged in to Firefox Relay page.
Relay add-on is installed.
The user is logged out of Instagram.

**Steps to reproduce:**

1. Navigate to https://www.instagram.com.
2. Refresh the page until Relay icon is displayed in the email field.
3. Observe the email field.

**Actual result:**
The email field lower line is not visible. The email placeholder text disappears. Same issue for Sign up page.
 
**Expected result:** 
The email field lower line is visible. The email placeholder text is visible.

**Note:**
The fact that the relay icon is displayed in the field only after Refresh might be another issue.

![email field issue instagram](https://user-images.githubusercontent.com/79697848/128168558-3ef955d2-4e30-4c42-9d9f-8202385c7dc5.PNG)
